index.js

```diff
@@ -239,6 +240,22 @@ async function main() {
       core.debug(inspect(data));
     }
 
+    if (inputs.team_reviewers) {
+      core.debug(`Adding team reviewers: ${inputs.team_reviewers}`);
+      const team_reviewers = inputs.team_reviewers.trim().split(/\s*,\s*/);
+      const { data } = await octokit.request(
+        `POST /repos/{owner}/{repo}/pulls/{pull_number}/requested_reviewers`,
+        {
+          owner,
+          repo,
+          pull_number: number,
+          team_reviewers
+        }
+      );
+      core.info(`Team reviewers added: ${team_reviewers.join(", ")}`);
+      core.debug(inspect(data));
+    }
```

Comment:
can you combine the `POST /repos/{owner}/{repo}/pulls/{pull_number}/requested_reviewers` request?